python/lsst/obs/decam/ingest.py

```diff
@@ -22,54 +22,23 @@
 import os
 import re
 
+from deprecated.sphinx import deprecated
 
-from lsst.daf.butler import ButlerURI, Formatter
 from astro_metadata_translator import fix_header, DecamTranslator
 from lsst.afw.fits import readMetadata
 from lsst.pipe.tasks.ingest import ParseTask, IngestTask, IngestArgumentParser
-from lsst.obs.base.ingest import RawFileData
 import lsst.obs.base
 
 __all__ = ["DecamRawIngestTask", "DecamIngestArgumentParser", "DecamIngestTask", "DecamParseTask"]
 
 
+@deprecated(reason="DECam no longer requires a specialist gen3 ingest task. Please use the default."
+            " Will be removed after v22.",
+            category=FutureWarning,
+            version="21.0")
```

Comment:
We've already released version 21, so `version` should be `"v22.0"`. That means the "will be removed" needs to be "after v23", barring special approval.